common/brave_content_client.cc

```diff
@@ -12,6 +12,52 @@
 #include "components/grit/components_resources.h"
 #include "content/public/common/url_constants.h"
 #include "ui/base/resource/resource_bundle.h"
+#include "third_party/widevine/cdm/buildflags.h"
+
+#if BUILDFLAG(ENABLE_WIDEVINE) && defined(OS_LINUX)
+#include "base/files/file_path.h"
+#include "base/files/file_util.h"
+#include "base/logging.h"
+#include "base/path_service.h"
+#include "chrome/common/chrome_paths.h"
+#include "chrome/common/media/component_widevine_cdm_hint_file_linux.h"
+#endif
+
+namespace {
+
+#if BUILDFLAG(ENABLE_WIDEVINE) && defined(OS_LINUX)
+// Hint file is used to give where widevine cdm library is installed.
+// On linux, zygote loads widevine before sandbox is initialized.
+// So, it should know where widevine is installed.
+// We will create hint file always if there is no hint file.
+// and that initial hint file will indicates widevine is installed to
+// user-dir/Widevine directory and it'll always work well.
+// If user don't install widevine yet, nothing is loaded.
+// If user installs old widevine, it's installed in user-dir/Widevine.
+// Then, newer version could be installed via component. In that case, hint
+// file will be updated to point to newer version. That new version will be
+// loaded after re-launching.
+void CreateDefaultWidevineCdmHintFile() {
+  base::FilePath hint_file_path;
+  CHECK(base::PathService::Get(chrome::FILE_COMPONENT_WIDEVINE_CDM_HINT,
```

Comment:
are you sure we should kill the browser?

Reply:
Yes, cdm lib should be loaded during the zygot initialization before sandboxing.

Reply:
It's linux-only constraint.